README.md

```diff
@@ -5,10 +5,24 @@ This is an [extension for CiviCRM](https://docs.civicrm.org/sysadmin/en/latest/c
 
 ## Getting Started
 
-1. Download the extension
+1. Download the extension in zip file
+    - Extract the zip file and named as sourcetracker
+    - Changed the directory of file to civicrm\wp-content\uploads\civicrm\ext
 2. Install the extension
-3. Create a custom field named `Source` (the name should be exactly same & case-sensitive)
-4. Include this field in any Profile (as type hidden)
+    - Navigate to the **Administer** > **System Setting** > **Extension**
```

Comment:
Add one more point above it – `Login to your CMS and navigate to the CiviCRM dashboard`

Reply:
ok sure